README.md

```diff
@@ -462,6 +462,22 @@ keys are delegated attributes' names and values are keys on synced data Hash. Th
 version of `delegate :name, to: :synced_data` which works with Hash reserved attributes names, like
 `:zip`, `:map`.
 
+## Persisting fetched objects
```

Comment:
Should we mention that it automatically enforces fetching data with `pagination with block`, regardless of `auto_paginate` value?